src/main/java/io/kestra/plugin/shopify/customers/CreateCustomer.java

```diff
@@ -0,0 +1,316 @@
+        title = "Last name",
+        description = "Last name of the customer"
+    )
+    private Property<String> lastName;
+
+    @Schema(
+        title = "Phone number",
+        description = "Phone number of the customer"
+    )
+    private Property<String> phone;
+
+    @Schema(
+        title = "Accepts marketing",
+        description = "Whether the customer accepts marketing emails"
+    )
+    @Builder.Default
+    private Property<Boolean> acceptsMarketing = Property.of(false);
+
+    @Schema(
+        title = "Verified email",
+        description = "Whether the customer's email has been verified"
+    )
+    @Builder.Default
+    private Property<Boolean> verifiedEmail = Property.of(false);
+
+    @Schema(
+        title = "Tax exempt",
+        description = "Whether the customer is tax exempt"
+    )
+    @Builder.Default
+    private Property<Boolean> taxExempt = Property.of(false);
+
+    @Schema(
+        title = "Tags",
+        description = "Comma-separated list of tags"
+    )
+    private Property<String> tags;
+
+    @Schema(
+        title = "Note",
+        description = "Additional note about the customer"
+    )
+    private Property<String> note;
+
+    @Schema(
+        title = "Password",
+        description = "Password for the customer account"
+    )
+    private Property<String> password;
+
+    @Schema(
+        title = "Password confirmation",
+        description = "Password confirmation for the customer account"
+    )
+    private Property<String> passwordConfirmation;
+
+    @Schema(
+        title = "Send email invite",
+        description = "Whether to send an email invite to the customer"
+    )
+    @Builder.Default
+    private Property<Boolean> sendEmailInvite = Property.of(false);
+
+    @Schema(
+        title = "Addresses",
+        description = "List of addresses for the customer"
+    )
+    private Property<List<AddressInput>> addresses;
+
+    @Schema(
+        title = "Metafields",
+        description = "Metafields for the customer"
+    )
+    private Property<Map<String, Object>> metafields;
+
+    @Data
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class AddressInput {
+        private String firstName;
+        private String lastName;
+        private String company;
+        private String address1;
+        private String address2;
+        private String city;
+        private String province;
+        private String country;
+        private String zip;
+        private String phone;
+        private String provinceCode;
+        private String countryCode;
+        private Boolean defaultAddress;
+    }
+
+    @Override
+    public Output run(RunContext runContext) throws Exception {
+        HttpClient client = buildHttpClient(runContext);
+        // Build customer object
+        Map<String, Object> customerData = new HashMap<>();
+        
+        String emailValue = runContext.render(email).as(String.class).orElse(null);
```

Comment:
You can remove the below check for null and do like 
```suggestion
        String emailValue = runContext.render(email).as(String.class).orElseThrow();
```

Reply:
sure, will do